docker/entrypoint.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [ "$1" = 'bety' ]; then
+    # Wait for Postgres to start
+    # http://www.onegeek.com.au/articles/waiting-for-dependencies-in-docker-compose
+    WAIT=0
+    while ! nc -z $PG_PORT_5432_TCP_ADDR $PG_PORT_5432_TCP_PORT; do
+      sleep 1
+      WAIT=$(($WAIT + 1))
+      if [ "$WAIT" -gt 15 ]; then
+        echo "Error: Timeout wating for Postgres to start"
+        exit 1
+      fi
+    done
+
+    # Move database config file into config directory & add host/port
+    cd /home/bety/config
+    /bin/cp /home/bety/docker/database.yml database.yml
+    /bin/sed -i "/host:/ s/$/ $PG_PORT_5432_TCP_ADDR/" database.yml
+    /bin/sed -i "/port:/ s/$/ $PG_PORT_5432_TCP_PORT/" database.yml
```

Comment:
Are these two lines really doing what you want them to do?  My sed version says the -i option is for providing an extension for renaming the unmodified version of the file.  Also, where do `PG_PORT_5432_TCP_ADDR` and `PG_PORT_5432_TCP_PORT` get set?  Is this a docker thing?